lib/org/openpsa/sales/handler/view.php

```diff
@@ -83,7 +83,7 @@ private function _populate_toolbar()
             );
         }
 
-        if ($this->_config->get('sales_pdfbuilder_class') && $this->_salesproject->can_do('midgard:update')) {
+        if ($this->_config->get('sales_pdfbuilder_class') && $this->_salesproject->can_do('midgard:update') && $this->is_pdf_creatable()) {
```

Comment:
pls put the conditions on separate lines, this is getting a bit too long to be readable